tests/onnxruntime/test_modeling.py

```diff
@@ -2312,15 +2314,14 @@ def test_compare_merged_and_not_merged_models_outputs(self, test_name: str, mode
         model_merged_dir = self.onnx_model_dirs[test_name + "_True"]
 
         model_not_merged = ORTModelForCausalLM.from_pretrained(model_not_merged_dir)
-        not_merged_onnx_path = Path(model_not_merged_dir, ONNX_DECODER_NAME)
+        not_merged_onnx_path = Path(model_not_merged_dir, ONNX_WEIGHTS_NAME)
         self.assertFalse(has_onnx_input(not_merged_onnx_path, "use_cache_branch"))
-        self.assertEqual(model_not_merged.use_merged, False)
+        self.assertFalse(model_not_merged.use_merged)
 
         model_merged = ORTModelForCausalLM.from_pretrained(model_merged_dir)
-        merged_onnx_path = Path(model_merged_dir, ONNX_DECODER_MERGED_NAME)
-        self.assertTrue(has_onnx_input(merged_onnx_path, "use_cache_branch"))
-        self.assertEqual(model_merged.decoder_with_past, None)
-        self.assertEqual(model_merged.use_merged, True)
+        merged_onnx_path = Path(model_merged_dir, ONNX_WEIGHTS_NAME)
+        self.assertFalse(has_onnx_input(merged_onnx_path, "use_cache_branch"))
+        self.assertFalse(model_merged.use_merged)
```

Comment:
?

Reply:
same !

Reply:
changed the logic in https://github.com/huggingface/optimum/pull/1257/commits/4a32f7a1c8ac7e0f6d7612eee503949277ce2b0f